ui/js/app.js

```diff
@@ -2,7 +2,10 @@ import store from "store.js";
 import lbry from "./lbry.js";
 
 const env = ENV;
-const config = require(`./config/${env}`);
+const config = {
+  ...require(`./config/${env}`),
+  appConfig: require("../../app/config.json"),
```

Comment:
So there are 2 things about this that could make it feel over-engineered:

1. `config.json` is a separate file, and it's at the app level, not the UI level
2. We're importing it into the Redux config object as a sub-object

#1 is because conceptually these are settings that apply to the whole app, even though currently we only have a single one that's used in the UI. Even with this one setting, later on there could easily be app level stuff that wants to know the UI version.

#2 is just to make it easy to access app-level settings in the UI without having to require `config.json` every time. The Redux config object seemed like a logical enough place.

Hopefully if we switch to using `package.json`, that will make it easy to access config settings within the render process, and we won't need this. Not sure yet.